pressbooks/pressbooks
# fix: specify domain translation for contributors imploding method

This PR intends to fix: https://github.com/pressbooks/pressbooks/issues/2359. Please read it for context.

We prioritize a logged in user language selection (locale) over the book's language when displaying a book from its home page. This change applies this prioritization only when the user is in the admin interface.

### Testing
- Log in using an admin or regular user.
- Go to Edit profile > Language and pick `English (United States)` for example, or any other, except the `Site Default` option.
- Pick a book and go to the Edit Info section
- Add two or more authors to the book, pick a language different from the language you picked up in your profile. For example, following the test case, you could choose `Spanish`. Make sure you add a license.
- Save the Book Info.
- Make the book public in Organize > Public option.
- Go to the book home page. You should see the list of authors with a `and` between the last two authors. It must be in the language of the book. License and other metadata must be in the Book Language, also.
- Log out and revisit the book home page.
- All the metadata, including `and` and license description, must be in the selected book language.
